lib/helpers/metaLoader.js

```diff
@@ -100,6 +100,21 @@ module.exports = class MetaLoader {
       });
   }
 
+  async getLinkedObjectsModel() {
+    return this.connection.describe(this.configuration.sobject)
+      .then(async (meta) => {
+        const model = {};
+        await meta.fields
+          .filter((field) => field.type === 'reference')
+          .forEach((field) => {
+            if (field) {
+              model[field.relationshipName] = `${field.referenceTo.join(', ')} (${field.relationshipName})`;
```

Comment:
Add some sort of check to make sure `referenceTo` is an array with a single element. (maybe)  (Throw error on zero.)